domains/authorization/Authorization.API/API/Authorization/[Features]/GetConsentForClientQuery.cs

```diff
@@ -22,21 +24,21 @@ public GetConsentForClientQueryHandler(IClientRepository clientRepository)
     public async Task<GetConsentForClientQueryResult> Handle(GetConsentForClientQuery query,
         CancellationToken cancellationToken)
     {
-        var idpClientID = new IdpClientId(query.ClientId);
+        var requestedClientId = new IdpClientId(query.ClientId);
 
-        var result = await _clientRepository.Query()
-            .Where(x => x.IdpClientId == idpClientID)
+        var foundClient = await _clientRepository.Query()
```

Comment:
minor: To me this name indicate a boolean - i suggest: client

Reply:
Done